api/models/photo.py

```diff
@@ -66,6 +68,8 @@ class Photo(models.Model):
 
     public = models.BooleanField(default=False, db_index=True)
     encoding = models.TextField(default=None, null=True)
+    clip_embeddings = ArrayField(models.FloatField(blank=True, null=True), size=512)
+    clip_embeddings_magnitutde = models.FloatField(blank=True, null=True)
```

Comment:
Typo: Should be  clip_embeddings_magnitude

Reply:
Fixed in commit be1dab2